env-defaults

```diff
@@ -251,11 +254,12 @@ STAGING_BUCKET=local-staging-bucket
 # Default: A bucket created for use in localstack
 ARTIFACT_BUCKET=local-artifact-bucket
 
+BACKEND_HOST=harmony
 # Development only: hostname to be used to connect to the host OS, used in
```

Comment:
Replace this old comment about different Linux and Mac OS X values and correctly document this parameter.

Reply:
Done